app/src/main/kotlin/com/mapbox/vision/teaser/replayer/ReplayModeFragment.kt

```diff
@@ -29,21 +30,32 @@ class ReplayModeFragment : Fragment(), OnBackPressedListener {
     }
 
     private lateinit var sessionsAdapter: SessionsAdapter
+    private lateinit var sessionsPath: String
+
+    private fun containsFiles(path: String) = File(path).listFiles().isNotEmpty()
```

Comment:
```suggestion
    private fun isFolderContainsFiles(path: String) = File(path).listFiles().isNotEmpty()
```
or
```suggestion
    private fun isDirectoryContainsFiles(path: String) = File(path).listFiles().isNotEmpty()
```

maybe also move to some utility class? if we have

Reply:
fixed